content/ja/docs/reference/glossary/kops.md

```diff
@@ -0,0 +1,26 @@
+---
+title: kOps (Kubernetes Operations)
+id: kops
+date: 2018-04-12
+full_link: /docs/setup/production-environment/kops/
+短い説明: >
+  kOps は、運用グレードの高可用性 Kubernetes クラスターの作成、破棄、アップグレード、維持を支援するだけでなく、必要なクラウド インフラストラクチャのプロビジョニングも行います。
+aka: 
+tags:
+- tool
+- operation
+---
+
+`kOps`は、運用グレードの高可用性 Kubernetes クラスターの作成、破棄、アップグレード、維持を支援するだけでなく、必要なクラウド インフラストラクチャのプロビジョニングも行います。
+
+<!--more--> 
+
+> **Note:** AWS (アマゾン ウェブ サービス) は現在正式にサポートされており、DigitalOcean、GCE、OpenStack はベータ版で、Azure はアルファ版でサポートされています。
+
+`kOps`は自動プロビジョニング システムです。
+  * 完全に自動化されたインストール
+  * DNS を使用してクラスターを識別します
+  * 自己修復: すべてが Auto Scaling グループで実行されます
```

Comment:
[NITS] Deleting whitespaces and replacing Hiragana with Kanji for a more natural translation.
```suggestion
  * 自己修復: 全てをAuto Scalingグループで実行
```